CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+# Video Headline - Contributing
+
+## Table of Contents
+
+- [Running the application in local environment](#running-the-application-in-local-environment)
+- [How to build the player](#how-to-build-the-player)
+- [How to run the tests](#how-to-run-the-tests)
+- [How to use the linters](#how-to-use-the-linters)
+
+### Running the application in local environment
+
+To set up the project locally, follow the instructions provided below.
+
+#### Prerequisites
+
+- AWS Account: Necessary for hosting and delivering video content.
+- Docker and Docker compose: Video Headline runs inside Docker containers, so it is necessary to have Docker and Docker Compose installed.
+- Yarn and Node.js(v10): Required to deploy AWS configurations and build the playerReact component.
+- Python: Necessary for running Django and other Python-based tools.
+- AWS CLI: Useful for configuring and managing AWS services from the command line.
+
+#### AWS Configuration
+
+Video Headline requires some IAM roles and permissiones. To automate the configuration, there's CDK code to create a Stack with all the requirements.
+
+To deploy this stack, follow this steps:
+
+1. Navigate to the infrastructure directory.
+2. Run the command: `yarn cdk deploy AwsConfigurationStack`.
+
+This deployment will set up:
+
+- Api User with permissions for:
+  - S3
+  - Sns
+  - MediaConvert
+  - MediaLive
+  - Cloudfront
+  - Cloudwatch
+- Media Convert Role with permissions for:
+  - Api Gateway
+  - S3
+- Media Live Role with permissions for:
+  - MediaLive
+  - Cloudwatch
+
+#### Set up the application in local environment
+
+To set up the project locally, follow the instructions provided below. For AWS deployment, refer to the README within the Infrastructure folder.
+
+**Environment Variables:** Add `AWS_MEDIA_CONVERT_ROLE` and `AWS_MEDIA_LIVE_ROLE` with respective ARNs to your Docker Compose file based on your environment (`docker-compose.dev.yml` or `docker-compose.prod.yml`).
```

Comment:
About this line

1. .env-example already contains AWS_MEDIA_CONVERT_ROLE and AWS_MEDIA_LIVE_ROLE, so i dont think we need to explicitly add it to the docker compose file
2. This is better suited for the section below `(Create .env file)`